modin/experimental/core/execution/unidist/implementations/pandas_on_unidist/io/sql.py

```diff
@@ -0,0 +1,275 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Module houses util functions for handling experimental SQL IO functions."""
+
+from collections import OrderedDict
+from sqlalchemy import MetaData, Table, create_engine, inspect
+
+
+def is_distributed(partition_column, lower_bound, upper_bound):
+    """
+    Check if is possible to distribute a query with the given args.
+
+    Parameters
+    ----------
+    partition_column : str
+        Column name used for data partitioning between the workers.
+    lower_bound : int
+        The minimum value to be requested from the `partition_column`.
+    upper_bound : int
+        The maximum value to be requested from the `partition_column`.
+
+    Returns
+    -------
+    bool
+        Whether the given query is distributable or not.
+    """
+    if (
+        (partition_column is not None)
+        and (lower_bound is not None)
+        and (upper_bound is not None)
+    ):
+        if upper_bound > lower_bound:
+            return True
+        else:
+            raise InvalidArguments("upper_bound must be greater than lower_bound.")
```

Comment:
```suggestion
        raise InvalidArguments("upper_bound must be greater than lower_bound.")
```

Reply:
let's leave this comment to be fixed in https://github.com/modin-project/modin/issues/5275, in order to simplify code deduplication.